assets/js/lesson_5/hw_11.js

```diff
@@ -0,0 +1,194 @@
+// 1.Show number from 20 to 30 with 0.5 step
+
+let firstTask = [];
+
+for (i = 20; i <= 30; i += 0.5) {
+    firstTask.push(i)
+}
+
+console.log(firstTask.join(','));
```

Comment:
dont forget about empty space. It looks better)
```suggestion
console.log(firstTask.join(', '));
```

Reply:
Missed this one )